test/ComposerRequireCheckerTest/NodeVisitor/UsedSymbolCollectorTest.php

```diff
@@ -289,4 +291,37 @@ public function testBeforeTraverseResetsRecordedSymbols(): void
 
         $this->assertCount(0, $this->visitor->getCollectedSymbols());
     }
+
+    public function testPropertyType(): void
+    {
+        $node       = new Property(Class_::MODIFIER_PUBLIC, []);
+        $node->type = new Name('Bar');
+
+        $this->visitor->enterNode($node);
+
+        $symbols = $this->visitor->getCollectedSymbols();
+        $this->assertCount(1, $symbols);
+        $this->assertContains('Bar', $symbols);
```

Comment:
```suggestion
        $symbols = $this->visitor->getCollectedSymbols();
                
        $this->assertCount(1, $symbols);
        $this->assertContains('Bar', $symbols);
```

Reply:
Is this only about spacing? If so, then let's leave it as is, and propose new CS rules in a separate patch.